CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Fix: `Sentry.close()` closes native SDK integrations (#388)
 * Fix: Mark `Sentry.currentHub` as deprecated (#406)
+* Feat: Support envelope based transport for events (#391)
```

Comment:
let's add this to a breaking change section, this also requires a newer Sentry server version.

See breaking change section https://github.com/getsentry/sentry-dart/releases/tag/5.0.0
See note about sentry server https://github.com/getsentry/sentry-dart/releases/tag/4.0.0
this was only for `sentry_flutter`, but now, also for `sentry`